arctic/serialization/numpy_strings.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import numpy as np
+import numpy.ma as ma
+import pandas as pd
+
+from bson import Binary, SON
+
+from .._compress import compress, decompress
+
+
+class NumpyArrayConverter(object):
+    """
+    Converts a Numpy ndarray to and from PyMongo SON representation:
+
+        {
```

Comment:
they are pretty similar. Since chunkstore is pretty new, so I'm free to change whatever I want without it affecting anyone. Tickstore obviously cannot be changed since its in use by many users. If I can get tickstore's serializer working in chunkstore I'll rip it out and make it shared between them.